Core/src/TrackFinding/CMakeLists.txt

```diff
@@ -1,6 +1,2 @@
-target_sources(
-  ActsCore
-  PRIVATE
-    CombinatorialKalmanFilterError.cpp
-    MeasurementSelector.cpp
-)
+target_sources(ActsCore PRIVATE CombinatorialKalmanFilterError
+                   .cpp MeasurementSelector.cpp FasTrackConnector.cpp)
```

Comment:
It looks like `clang-format` was run on this file, as well as the 2 modified `CMakeLists.txt` files, `reconstruction.py`, and `full_chain_itk.py` files. I'm afraid this will break them.
Sorry, I should have mentioned that `clang-format` is just for C++ files (`.cpp`, `.hpp`, `.ipp`). Can you revert the last change for the non-C++ files?